src/main/java/com/sapred/ordermanagerred/service/UserService.java

```diff
@@ -20,53 +28,75 @@ public class UserService {
     private UserRepository userRepository;
 
     @Autowired
-    private AddressRepository addressRepository;
+    private JwtToken jwtToken;
 
     @Autowired
-    private JwtToken jwtToken;
+    private RoleRepository roleRepository;
+
+    @Autowired
+    CompanyRepository companyRepository;
 
-    // שימי לב: זו סתם פונקציה שמכניסה נתונים בשביל הבדיקה
+//     שימי לב: זו סתם פונקציה שמכניסה נתונים בשביל הבדיקה
     public void fill() {
-        AuditData d = new AuditData( new Date(),new Date());
-        Role roles = new Role("11", RoleOptions.CUSTOMER, "kkkR",d);
-        Company c = new Company("52","gg",55,d);
-        Address a = new Address( "0580000000","mezada 7","kamatek@gmail.com");
-        addressRepository.save(a);
-        User user = new User("123","user","mypass",a,roles,c,d);
+        AuditData d = new AuditData(new Date(), new Date());
+        Role roles = new Role("1", RoleOptions.CUSTOMER, "cust", d);
+        roleRepository.save(roles);
+        Company c = new Company("1", "osherad", 55, d);
+        companyRepository.save(c);
+        Address a = new Address("0580000000", "mezada 7", "emailCust@gmail.com");
+        User user = new User("4", "new full name", "passCust", a, roles, c, d);
         userRepository.save(user);
     }
 
-    public List<User> getAll() {
-        List<User> listAll = userRepository.findAll();
-        return listAll;
-    }
-
-
     public ResponseEntity<String> logIn(String email, String password) {
-        User authenticatedUserEmail = authenticateUserEmail(email);
+        User authenticatedUserEmail = userRepository.getByAddressEmail(email);
         if (authenticatedUserEmail == null)
             return new ResponseEntity<>("Resource not found", HttpStatus.NOT_FOUND); // 404
```

Comment:
In the Service throw UserNotFound and do not return the rest code HttpStatus.NOT_FOUND